src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,206 @@
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, driverId);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                Car car = parseCar(resultSet);
+                car.setManufacturer(parseManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get all cars by driverId + " + driverId, e);
+        }
+        cars.forEach(c -> c.setDrivers(getDriversFromDB(driverId)));
+        return cars;
+    }
+
+    private Manufacturer parseManufacturer(ResultSet resultSet) throws SQLException {
+        Long id = resultSet.getObject("manufacturer_id", Long.class);
+        String name = resultSet.getString("manufacturer_name");
+        String country = resultSet.getString("manufacturer_country");
+        return new Manufacturer(id, name, country);
+    }
+
+    private Car parseCar(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setId(resultSet.getObject("id", Long.class));
+        car.setModel(resultSet.getString("model"));
+        return car;
+    }
+
+    private List<Driver> getDriversFromDB(Long id) {
+        List<Driver> drivers = new ArrayList<>();
+        String query = "SELECT id, name, license_number FROM drivers "
+                + "JOIN cars_drivers cd ON drivers.id = cd.driver_id WHERE cd.car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                drivers.add(parseDriver(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get drivers by id " + id, e);
+        }
+        return drivers;
+    }
+
+    private Driver parseDriver(ResultSet resultSet) throws SQLException {
+        Long id = resultSet.getObject("id", Long.class);
+        String name = resultSet.getString("name");
+        String licenseNumber = resultSet.getString("license_number");
+        return new Driver(id, name, licenseNumber);
+    }
+
+    private void deleteRelationship(Car car) {
+        String query = "DELETE FROM cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, car.getId());
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't delete car: " + car, e);
+        }
+    }
+
+    private void addDrivers(Car car) {
```

Comment:
method should have understandable name 
```suggestion
    private void addDriversToCar(Car car) {
```